docs/3.api/1.composables/use-async-data.md

```diff
@@ -34,11 +34,14 @@ type AsyncData<DataT, ErrorT> = {
   refresh: (opts?: AsyncDataExecuteOptions) => Promise<void>
   execute: (opts?: AsyncDataExecuteOptions) => Promise<void>
   error: Ref<ErrorT | null>
+  status: Ref<AsyncDataRequestStatus>
 };
 
 interface AsyncDataExecuteOptions {
   dedupe?: boolean
 }
+
+type AsyncDataRequestStatus = 'IDLE' | 'PENDING' | 'SUCCESS' | 'ERROR'
```

Comment:
would you mind making these lowercase?

Reply:
Just for consistency - our other string literal types are lowercase.

Reply:
Makes sense, thanks.